astropy/wcs/setup_package.py

```diff
@@ -16,7 +16,7 @@
 from extension_helpers import import_file, write_if_different, get_compiler, pkg_config
 
 WCSROOT = os.path.relpath(os.path.dirname(__file__))
-WCSVERSION = "7.3.0"
+WCSVERSION = "7.4"
```

Comment:
Consistency with what do we want? I've been looking at previous versions and found that only a single one of recent WCSLIB releases had a minor version, 7.3.1. So I would rather prefer consistency with the version the official release defines (`PACKAGE_VERSION='7.4'` in configure), unless there are strong reasons for having a 3-part version number somewhere else.

Reply:
Basically keeping it the old style would mean that `wcs._wcs.__version__` returned '7.4' if built with `ASTROPY_USE_SYSTEM_WCS=1`, and '7.4.0' otherwise. Of course we could take this as a bug or a feature. 😃

Reply:
We've been using 3 digit versions in astropy since astropy 3.0 and I'd like to be self-consistent.
In the past wcslib has omitted the 3rd digit when it is `0` and used it when it's something else and that's not a good practice.

Reply:
That probably went largely unnoticed since there were only 5 of the last 30 releases or so that had a minor version.

But OK with me, if that is preferable to avoiding different version styles (using the external library is to become the recommended version, iirc).